src/containers/IdentitySettingsContainer.ts

```diff
@@ -19,11 +20,24 @@ export const mapStateToProps = (state: AppState, ownProps: { navigation: TypedNa
 
 const mapDispatchToProps = (dispatch: any): DispatchProps => {
     return {
-        onUpdateAuthor: (text: string) => {
+        onUpdateAuthor: (text: string, ownFeed?: LocalFeed) => {
             dispatch(Actions.updateAuthorName(text));
+            if (ownFeed != null) {
```

Comment:
This looks fragile, it should be triggered by one action

Reply:
Changed